app/CMakeLists.txt

```
@@ -67,17 +67,22 @@ if ((NOT CONFIG_ZMK_SPLIT) OR CONFIG_ZMK_SPLIT_BLE_ROLE_CENTRAL)
 
   if (CONFIG_ZMK_BLE)
     target_sources(app PRIVATE src/events/ble_active_profile_changed.c)
+    
     target_sources(app PRIVATE src/behaviors/behavior_bt.c)
     target_sources(app PRIVATE src/ble.c)
     target_sources(app PRIVATE src/hog.c)
+    if (CONFIG_ZMK_SPLIT_BLE_ROLE_CENTRAL)
```

We moved to the more agnostic `CONFIG_ZMK_SPLIT_ROLE_CENTRAL` now:

```suggestion
    if (CONFIG_ZMK_SPLIT_ROLE_CENTRAL)
```